label_studio_converter/converter.py

```diff
@@ -610,10 +611,42 @@ def convert_to_yolo(self, input_data, output_dir, output_image_dir=None, output_
                 category_id = category_name_to_id[category_name]
 
                 if "rectanglelabels" in label or 'labels' in label:
-                    x = (label['x'] + label['width']/2) / 100
-                    y = (label['y'] + label['height']/2) / 100
-                    w = label['width'] / 100
-                    h = label['height'] / 100
+                    label_x, label_y, label_w, label_h, label_r = (
+                        label["x"],
+                        label["y"],
+                        label["width"],
+                        label["height"],
+                        label["rotation"],
+                    )
+                    if abs(label_r) > 1:
```

Comment:
why not `abs(label_r) > 0` ?

Reply:
`abs(label_r) > 0` is ok, even there can be no judgment. But considering the possibility of mistaken rotation and the angle after recovery will not be zero, and to avoid unnecessary calculations, `abs(label_r) > 1` is better.

Reply:
I would use > 0, because for huge boxes 0.5 will be a significant movement.

Reply:
Gotcha